src/components/modal/Modal.component.tsx

```diff
@@ -0,0 +1,76 @@
+import { useNavigate } from 'react-router-dom';
+
+import CloseIcon from '@mui/icons-material/Close';
+import DialogActions from '@mui/material/DialogActions';
+import DialogContent from '@mui/material/DialogContent';
+import IconButton from '@mui/material/IconButton';
+import Typography from '@mui/material/Typography';
+
+import profile from '@assets/vectors/Avatar.svg';
+import { CustomButton } from '@components';
+import { ROUTES } from '@constants';
+import { t } from '@localization';
+
+import {
+    BootstrapDialog,
+    ContentWrapper,
+    ImageWrapper,
+    StyledImage,
+} from './Modal.styles';
+
+interface ProfileModalProps {
+    open: boolean;
+    handleClose: () => void;
+}
+
+export const ProfileModal = ({ open, handleClose }: ProfileModalProps) => {
+    const navigate = useNavigate();
+    const handleLogout = () => {
+        localStorage.removeItem('isAuthenticated');
+        void navigate(ROUTES.LOGIN);
+    };
+    return (
+        <BootstrapDialog
+            onClose={handleClose}
+            aria-labelledby="customized-dialog-title"
```

Comment:
this code is directly copied from the docs line for line (https://mui.com/material-ui/react-dialog/#customization)

<img width="882" height="622" alt="Image" src="https://github.com/user-attachments/assets/ece6b086-1ee8-452c-a642-ee0cea568f4e" />

discuss once.